results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/DutchBookReviewSentimentClassification.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dataset_revision": "3f756ab4572e071eb53e887ab629f19fa747d39e",
+  "mteb_dataset_name": "DutchBookReviewSentimentClassification",
+  "mteb_version": "1.6.10",
+  "test": {
+    "accuracy": 0.5443794964028777,
```

Comment:
The performance looks quite similar for the two models - just to ensure that there is a signal in the data will you run the e5-multilingual-base as well?

Reply:
Wow it's much better.